Adadie/Fresh
# HTML & CSS UI Features

#### What does this PR do?
Contains HTML & CSS UI to be merged to develop branch
#### Description of Task to be completed?
Have the following pages working:

- [x] Login Page
- [x] Sign Up page
- [x] Blogs Page
- [x] Bio Page
- [x] Experience Page
- [x] Contact Page

#### How should this be manually tested?

- Clone repository to your local directory [click here to get the link](https://github.com/Junior-Adadie/Fresh.git)
- Open index.html file with your browser
- Navigate through all the pages
- Check if the footer and header are fixed
- Test all buttons and input boxes  to see if they work properly
- Also, reduce the screen scale up to 768 pixels to test its responsiveness on a tablet or mobile phone
#### Any issues or blockers in the project
Profiles page is still in working progress
#### What are the relevant Trello stories?
[Trello Board](https://trello.com/b/7VHyWhdz/ka-blog)
#### Any supporting screenshots and images?
<img width="480" alt="clone-button-url-mac" src="https://user-images.githubusercontent.com/68109866/88227421-41c72980-cc76-11ea-9eaf-d84a4abeaa58.png">

![maxresdefault](https://user-images.githubusercontent.com/68109866/88227465-53a8cc80-cc76-11ea-81f4-dd7d28dadbc3.jpg)

![copy-clone-url](https://user-images.githubusercontent.com/68109866/88230955-02034080-cc7c-11ea-8637-8572b333a497.png)
![drag-and-drop-file_chrome](https://user-images.githubusercontent.com/68109866/88231024-2101d280-cc7c-11ea-8cac-f53aa753260d.png)
